recipes/datview/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "datview" %}
+{% set version = "1.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/datview-{{ version }}.tar.gz
+  sha256: cfc6b19ea6c82e98a47bb43da08216bd0c9fde4ca058d523cb17079448d210d0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  entry_points:
+    - datview = datview.main:main
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+    - tk  
+
+  run:
+    - python >={{ python_min }}
+    - h5py
+    - hdf5plugin
+    - pillow
+    - matplotlib
```

If it is OK for the tests, it may be OK here too, right?

```suggestion
    - matplotlib-base
```